### ISSUE TITLE
Can issue #574 also have a "Subgroup: Visual Contrast" label applied to it?

### ISSUE THREAD

**Issue by ericwbailey** [w3c/silver#724]
It appears the "[Subgroup: Visual Contrast](https://github.com/w3c/silver/issues?q=is%3Aopen+label%3A%22Subgroup%3A+Visual+Contrast%22+sort%3Aupdated-desc)" label is missing in https://github.com/w3c/silver/issues/574, despite dealing with color contrast. It appears similar to https://github.com/w3c/silver/issues/272 and https://github.com/w3c/silver/issues/651, which led me to consider it a candidate for it being applied.

Personally, I feel the label is important for context for interested parties, as well as the historical record for the decision-making process.

**Comment by rachaelbradley** [comment]
This is complete.
